### ISSUE TITLE
Select control in new onboarding experience is not keyboard navigable

### ISSUE THREAD

**Issue by JakePT** [woocommerce/woocommerce-admin#3877]
**Describe the bug**
When setting up a store with the new onboarding screen in 4.0, when you are selecting a country you cannot move between options with the up and down arrows.

**To Reproduce**
1. Install WooCommerce
2. Begin setup Wizard
3. Select Country/Region dropdown, type a few letters
4. Use up and down arrows to try and move between options

**Expected behavior**
I can move between options with the arrow keys.

**Isolating the problem (mark completed items with an [x]):**
- [x ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x ] I can reproduce this bug consistently using the steps above.

**Comment by psealock** [comment]
While looking into this, we should consider using https://github.com/WordPress/gutenberg/tree/master/packages/components/src/custom-select-control

**Comment by mattsherman** [comment]
@JakePT Thanks for reporting. I am unable to reproduce this. Can you please provide some more information on the browser environment you are experiencing this in?

- OS
- Browser
- Browser extensions

**Comment by JakePT** [comment]
I could not replicate the issue in Chrome on MacOS WooCommerce 4.0.1 with no browser extensions. 

I originally experienced the issue with Chrome on Windows 10 with WooCommerce 4.0.0. I don't have access to Windows right now to test again (🦠).

**Comment by mattsherman** [comment]
I tested this on Windows 10 with both Chrome and Edge (legacy) and could not reproduce this. Closing the issue.
